assignments/arrays.js

```diff
@@ -63,34 +75,96 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
+function findObj(arr, id) {
+  for (const x of arr) {
+    if (x["id"] == id) {
+      return x;
+    }
+  }
+}
 
+const res = findObj(inventory, 33);
+console.log(`Car 33 is a ${res["car_year"]} ${res["car_model"]}`);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory[inventory.length - 1];
+console.log(`${lastCar.car_year} and ${lastCar.car_model}`);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+
```

Comment:
damn son. You wrote a lot of code for this. I'm impressed. You could have done it potentially easier but I like the way you did it because your answer comes out looking cleaner. Well done.